docs/lib/restapi/fragments/js/fetch.md

```diff
@@ -70,17 +70,17 @@ API
 Example with async/await:
 
 ```javascript
-async function head() { 
+function head() { 
     const apiName = 'MyApiName';
     const path = '/path';
     const myInit = { // OPTIONAL
         headers: {}, // OPTIONAL
     };
 
-    return await API.head(apiName, path, myInit);
+    return API.head(apiName, path, myInit);
 }
 
-head();
+const response = await head();
```

Comment:
Top-level await is unsupported in most environments. Otherwise, good fixes! Surprised these were in here to begin with :)

Reply:
Sure, I was just assuming that call would be inside another function. (the docs could be modified to show that, maybe)